package.json

```diff
@@ -57,7 +57,7 @@
     "lodash.union": "^4.6.0",
     "lodash.uniqby": "^4.4.0",
     "react": "16.13.1",
-    "react-intl": "5.8.6",
+    "react-intl": "5.20.1",
```

Comment:
Did you test using the app in a different language?

Reply:
Yes; I'll make a note that I did, in the commit message.